src/config/2021.json

```diff
@@ -650,6 +670,14 @@
       "github": "philbarker",
       "twitter": "philbarker"
     },
+    "RMHolmlund": {
+      "name": "RMHolmlund",
+      "teams": [
+        "editors"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/61608378?v=4&s=200",
+      "github": "RMHolmlund"
+    },
```

Comment:
@RMHolmlund do you want your full name credited in the chapter? Also let us know if there are any other platforms you'd like promoted (Twitter, LinkedIn, or personal website)